Services/ServiceService.cs

```diff
@@ -0,0 +1,31 @@
+using HairdresserAPI.Domains.HairdresserDomain.DTO;
+using HairdresserAPI.HairdresserDomain.Aggregate;
+using HairdresserAPI.Interfaces;
+
+namespace HairdresserAPI.Services;
+
+public class ServiceService : IServiceService
+{
+    private readonly IServiceRepository _serviceRepository;
+
+    public ServiceService(IServiceRepository serviceRepository)
+    {
+        _serviceRepository = serviceRepository;
+    }
+
+    public async Task<Service> AddService(ServiceDto serviceDto)
+    {
+        var service = new Service(serviceDto.Name, serviceDto.Description, serviceDto.Price, serviceDto.Duration, serviceDto.HairdresserId);
+
+        await _serviceRepository.AddService(service);
+
+        return service;
```

Comment:
After adding a service, you return the domain entity instead of a DTO. It's a common practice to return a DTO to abstract away the domain model from the API consumers. So it should be something like this:
 
```
 public async Task<ServiceResponseDto> AddService(ServiceRequestDto serviceDto)
 {
     // create service
     var serviceResponseDto = await _serviceMappingService.MapServiceResponseDto(service);
     return serviceResponseDto;
 }
```